java/se/sics/mspsim/emulink/EmuLink.java

```diff
@@ -357,10 +357,4 @@ public void run() {
             System.exit(-1);
         }
     }
-
-    public static void main(String[] args) {
-        EmuLink el = new EmuLink();
-        el.run();
-    }
-
```

Comment:
This main method is currently the only remaining option to start EmuLink.

Reply:
Removed the file instead.